tests/js/jest.config.json

```diff
@@ -30,7 +30,9 @@
 		"@woocommerce/blocks-test-utils": "tests/utils",
 		"@woocommerce/types": "assets/js/types",
 		"@woocommerce/utils": "assets/js/utils",
-		"@woocommerce/interactivity": "assets/js/interactivity"
+		"@woocommerce/interactivity": "assets/js/interactivity",
+		"^react$": "<rootDir>/node_modules/react",
```

Comment:
These 2 lines are to ensure we only resolve one version of React. In prod this is never an issue because we load React via the window, but here we end up with multiple React instances sometimes in test and this causes "invalid hook" errors that fail tests.